documentation/pdb_client_tools.markdown

```diff
@@ -48,11 +52,35 @@ certificate-whitelist and specify the paths to the CLI node's cacert, cert, and
 private key when using the CLI either with flags or a configuration file.
 
 To configure the PuppetDB CLI to talk to your PuppetDB with flags, add a
-configuration file at `$HOME/.puppetlabs/client-tools/puppetdb.conf`. For more
+configuration file at `$HOME/.puppetlabs/client-tools/puppetdb.conf` (or
+`$USERPROFILE/.puppetlabs/client-tools/puppetdb.conf` for Windows). For more
 details see the installed man page:
 
     $ man puppetdb_conf
 
+The PuppetDB CLI configuration files (the user-specified or global files) can
+take the following settings:
+
+- `server_urls` Either a JSON String (for a single url) or Array (for multiple
+  urls) of your PuppetDB servers to query or manage via the CLI commands. (You
+  can set this with the `puppetdb_urls` parameter in the
+  puppet_enterprise::profile::controller class for PE.)
+
+  Default value: https://127.0.0.1:8080
+
+- `cacert` The path for the CA cert.
+
+  *nix sytems - /etc/puppetlabs/puppet/ssl/certs/ca.pem
+  Windows - C:/ProgramData/PuppetLabs/client-tools/ssl/certs/ca.pem
+  
+- `cert` An SSL certificate signed by your site's Puppet CA. Note that the PE
+ version of the CLI supports token auth via `puppet-access` and this option
+ should not be necessary.
+
+- `key` The private key for that certificate. Note that the PE version of the
+ CLI supports token auth via `puppet-access` and this option should not be
+ necessary.
+
```

Comment:
In all the other config docs for various client tools, we list the options and then have a JSON example: https://docs.puppet.com/pe/latest/code_mgr_cli.html#installing-and-configuring-puppet-code-on-agents-and-workstations